Dockerfile

```diff
@@ -38,3 +38,9 @@ RUN apt-get -qqy update && apt-get install -qqy \
     docker --version && kubectl version --client
 VOLUME ["/root/.config", "/root/.kube"]
 
+# Expose the Datastore port
+EXPOSE 8082
+
+
+# Set the default command to run when starting the container
+CMD ["/bin/bash", "-c", "gcloud beta emulators datastore start   --no-store-on-disk   --project=coder-cr   --host-port 0.0.0.0:8082"]
```

Comment:
re: `--project=coder-cr`

Can I reuse this container for my project. Do I need to start another emulator with my project name?

Reply:
I believe this option for the emulator is actually ignored. Let me experiment with removing it and confirm tests still build + work. We should be able to use the same emulator image across projects, at least right now when we don't apply seed data when starting up the emulator.

Getting seed data into the emulator is sort of a different discussion; I was having trouble getting data files to export from the emulator in a manner where they could later be imported to a fresh emulator (see docs here: https://cloud.google.com/datastore/docs/tools/emulator-export-import) and am not sure the solution to this. If in the future we have different seed data on a per-project basis which needs to be added in on the container build step, this could necessitate separate emulators per-project.

Reply:
Confirmed removing project flag does not have impact on tests. Updated to remove!

Reply:
I'm having trouble running the container

```
tai@DESKTOP-7R2MMFM:~/git/rscore$ docker run \
> -it \
> --rm \
> -p 8082:8082  \
> gcr.io/cbio-cicd/cerebrae/google/cloud-sdk-docker
ERROR: (gcloud.beta.emulators.datastore.start) The required property [project] is not currently set.
You may set it for your current workspace by running:

  $ gcloud config set project VALUE

or it can be set temporarily by the environment variable [CLOUDSDK_CORE_PROJECT]
```

Reply:
Did you run it locally or in Cloud Build? Can you verify that your build ran with the latest image, 3fa6f82?